testcases/OpTestHMIHandling.py

```diff
@@ -134,8 +86,7 @@ def test_init(self):
                 else:
                     l_cores[int(matchObj.group(1))] = list(matchObj.group(2))
         if not l_cores:
-            l_msg = "Failed in getting core id's information from OPAL msg log"
-            raise OpTestError(l_msg)
+            raise Exception("Failed in getting core ids information from OPAL msg log")
 
```

Comment:
Can we change the above command l_cmd to "cat /sys/firmware/opal/msglog | grep -i --color=never CHIP" as it fails to gather the core id's due to the change in ssh execution method, where it is grep's with color